### DISCUSSION TITLE
Submit Public Block List

### DISCUSSION THREAD

**Discussion by IgnoreBox** [uBlockOrigin/uBlock-issues#2318] [Q&A] [answered]
Hello. how to submit my own public block list ?

**Comment by ghost** [comment, marked as answer]
Create it, extend, maintain. If you don't get bored year later, then submit it to [`filterlists.com` database](https://github.com/collinbarrett/FilterLists/issues). When it will be useful and people will use it, it will be eventually noticed and maybe will be added to uBO if it will be good fit and will have good quality filters.

**Reply by IgnoreBox** [reply]
ok thankyou
